packages/lds/src/HorizonScroller/HorizonScroller.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { useLayoutEffect, useRef } from 'react';
+
+import { horizonScroller } from './HorizonScroller.css';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+const ChipGroup = ({ children }: Props) => {
+  const ref = useRef<HTMLDivElement | null>(null);
```

Comment:
[trivial]
```suggestion
  const ref = useRef<HTMLDivElement>(null);
```
useRef에 null이 기본적으로 타입정의 되어 있어서 제거해도 괜찮을것 같네요!

Reply:
https://github.com/YAPP-Github/23rd-Web-Team-1-FE/pull/33/commits/e1b1d172f7aa295d9792ce0a86321a21f99e2f92 수정했어융